erpnext/crm/doctype/lead/lead.js

```diff
@@ -88,14 +88,14 @@ erpnext.LeadController = class LeadController extends frappe.ui.form.Controller
 	make_customer() {
 		frappe.model.open_mapped_doc({
 			method: "erpnext.crm.doctype.lead.lead.make_customer",
-			frm: this.frm,
+			frm: cur_frm,
```

Comment:
`cur_frm` should strictly be used for browser console.

Reply:
I based this on sales_invoice.js, delivery_note.js, and purchase_receipt.js because it is used in a similar way in those files.